docs-ref-conceptual/run-azure-cli-docker.md

```diff
@@ -45,8 +50,10 @@ Available versions can be found at [Azure CLI release notes](./release-notes-azu
 
 To run a specific version of the Azure CLI in the Docker container, use this command:
 
+**Discussion point #3:** Need link from engineering. How does the customer combine a tag with a specific azCLI version?
+
```

Comment:
You can find all tags here: https://mcr.microsoft.com/en-us/product/azure-cli/tags. So the command looks like:

```
docker pull mcr.microsoft.com/azure-cli:2.62.0-cbl-mariner2.0
```